src/Shopsys/ShopBundle/Model/Administrator/AdministratorFacade.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ShopBundle\Model\Administrator;
+
+use Shopsys\FrameworkBundle\Model\Administrator\Administrator as BaseAdministrator;
+use Shopsys\FrameworkBundle\Model\Administrator\AdministratorFacade as BaseAdministratorFacade;
+
+class AdministratorFacade extends BaseAdministratorFacade
+{
+    /**
+     * @param \Shopsys\ShopBundle\Model\Administrator\Administrator $administrator
+     */
+    protected function checkForDelete(BaseAdministrator $administrator)
+    {
+        parent::checkForDelete($administrator);
+
+        if ($administrator->isDefaultAdmin()) {
+            throw new \Shopsys\ShopBundle\Model\Administrator\Exception\DeletingDefaultAdminException();
```

Comment:
Trying to delete this admin cause error 500 in a contrast with for example deleting last admin, where nice flash message is shown.

Reply:
Well I have solved it same way as is deleting superadmin - that will end with 500 as well. Again you have no direct link in admin to delete this user, so I think it is ok.